records_mover/creds/creds_via_env.py

```diff
@@ -13,19 +13,41 @@
 
 
 class CredsViaEnv(BaseCreds):
-    def _gcp_creds(self, gcp_creds_name: str,
-                   scopes: Iterable[str]) -> 'google.auth.credentials.Credentials':
+    def _gcp_creds_from_env(self,
+                            scopes: Iterable[str]) ->\
+            Optional['google.auth.credentials.Credentials']:
+        if 'GCP_SERVICE_ACCOUNT_JSON_BASE64' not in os.environ:
+            return None
         import google.oauth2.service_account
-        notes_json_base64 = os.environ['GCP_SERVICE_ACCOUNT_JSON_BASE64']
-        notes_json = base64.b64decode(notes_json_base64.encode('utf-8'))
-        cred_details = json.loads(notes_json)
+        json_base64 = os.environ['GCP_SERVICE_ACCOUNT_JSON_BASE64']
+        json_str = base64.b64decode(json_base64.encode('utf-8'))
+        cred_details = json.loads(json_str)
 
         return google.oauth2.service_account.Credentials.\
             from_service_account_info(cred_details, scopes=scopes)
 
+    def _gcp_creds(self, gcp_creds_name: str,
+                   scopes: Iterable[str]) -> 'google.auth.credentials.Credentials':
+        # No current way to differentiate between different creds;
+        # just on env variable recognized.
+        creds_from_env = self._gcp_creds_from_env(scopes)
+        if creds_from_env is None:
+            raise KeyError('Please set GCP_SERVICE_ACCOUNT_JSON_BASE64 to configure Google '
+                           'Cloud Platform credentials')
+        return creds_from_env
```

Comment:
Note that in practice our internal use on laptops uses `CredsViaLastPass`, but this class gets used in places like JupyterHub and CircleCI.